packages/gatsby/src/internal-plugins/load-postcss-config/gatsby-node.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+const findPostcssPlugins = require(`postcss-load-plugins`)
+const autoprefixer = require(`autoprefixer`)
+const flexbugs = require(`postcss-flexbugs-fixes`)
+const report = require(`gatsby-cli/lib/reporter`)
+
+// { stage, rules, loaders, actions, store, getConfig }
+exports.onCreateWebpackConfig = async ({ actions, store }) => {
+  const program = store.getState().program
+  const { directory, browserslist } = program
+
+  const defaultPlugins = {
+    plugins: [
+      flexbugs,
+      autoprefixer({ browsers: browserslist, flexbox: `no-2009` }),
```

Comment:
It looks like we should check for a global browserslist config before using Gatsby's default config: https://github.com/postcss/autoprefixer#no-prefixes-in-production

Reply:
What's a global browserlist config? We should double-check we're loading the browserlist from all possible places.

Reply:
It looks like it should be a `.browserslistrc` file or `browserslist` key in `package.json` which should be present in the root of a project.

Reply:
i think we should probably limit this to just Gatsby's api and not use a .browserlistrc? otherwise we'd have to support the browserlist thing for all cases where we use the browsers list.

Reply:
We could use this tool https://github.com/browserslist/browserslist

I think it'd be good to support additional ways of specifying browserlist otherwise people will get confused. Right now we only look at the package.json but it'd be easy to swap in the above package there. https://github.com/gatsbyjs/gatsby/blob/b4aefcc8a0595c8e5bf0d6031e66ee2231bee4f9/packages/gatsby-cli/src/create-cli.js#L25